substrate/frame/revive/src/vm/evm/instructions/contract.rs

```diff
@@ -121,36 +127,26 @@ pub fn call<'ext, E: Ext>(context: Context<'_, 'ext, E>) {
 		return;
 	};
 
-	let Some(account_load) = context.host.load_account_delegated(to) else {
-		context.interpreter.halt(InstructionResult::FatalExternalError);
-		return;
-	};
+	// TODO: Handle the cold/warm storage correctly
+	let scheme = CallScheme::Call;
+	let input = CallInput::SharedBuffer(input);
```

Comment:
We won't that will probably be baked in into frame if we do by checking whether or not the accessed storage is in the cache

nit: let's inline this as in the original code

Reply:
you mean to inline scheme and input? I am using it now also in calc_call_gas not only in action